packages/rest/src/validation/request-body.validator.ts

```
@@ -93,19 +93,22 @@ function validateValueAgainstJsonSchema(
   const ajv = new AJV({
     allErrors: true,
   });
+  let errors;
   try {
     if (ajv.validate(schemaWithRef, body)) {
       debug('Request body passed AJV validation.');
       return;
     }
   } catch (err) {
     debug('Cannot execute AJV validation: %s', util.inspect(err));
+    errors = [err];
```

Can we simply rethrow the error here?